AfsprakenRegels.md

```diff
@@ -781,6 +781,8 @@ Er kan sprake zijn van 1 naam en/of definitie die voor meerdere modelelementen g
 
 Echter, voor bijvoorbeeld voor een relatiesoort of een attribuutsoort kan het ook nodig zijn om dezelfde naam en/of definitie te specificeren. Dit is mogelijk, maar het is dan niet geheel duidelijk of er hetzelfde bedoeld wordt, of dat de (exacte) overeenkomt in een informatiemodel (zoals bv. in een UML informatiemodel) een toevalligheid is. Daarom gelden de volgende afspraken: 
 
+* Als de **uri** van twee of meer modelelementen EXACT hetzelfde is, dan wordt hiermee bedoeld dat het hetzelfde kenmerk is. Bijvoorbeeld bij objecttype Gebied met attribuutsoort identificatie met _uri_ "http://modellen.geostandaarden.nl/def/nen3610#identificatie" en een objecttype Boom met een attribuutsoort identificatie met _uri_ "http://modellen.geostandaarden.nl/def/nen3610#identificatie".
+
 * Als de **definitie** van twee of meer modelelementen EXACT hetzelfde is, dan wordt hiermee bedoeld dat het dezelfde definitie is. Bijvoorbeeld: attribuutsoort overboeking met _definitie_ "Het bedrag in euro's" en attribuutsoort koopsom met _definitie_ "Het bedrag in euro's".
 
 * Als de **naam** _en_ de **definitie** van twee of meer modelelementen EXACT hetzelfde is, dan wordt hiermee bedoeld dat het hetzelfde kenmerk is. Bijvoorbeeld bij objecttype Huis met attribuutsoort oppervlakte: : "De oppervlakte in hele vierkante meters" en objecttype Appartement met attribuutsoort oppervlakte : "De oppervlakte in hele vierkante meters".
```

Comment:
Eigenlijk klopt dit niet. Want zo lijken modelelement en kenmerken verschillende dingen, dwz: dan lijkt een kenmerk geen modelelement... Is het in dit geval namelijk hetzelfde attribuuttype? Of kan dat niet? Hebben we dit nu eigenlijk nog wel nodig? Want met URI's kunnen we dit ook al aangeven, en de URI is altijd aanwezig (expliciet ingevuld of af te leiden)